schemas/gen_ai.yml

```
@@ -0,0 +1,140 @@
+---
+- name: gen_ai
+  title: Gen AI
+  short: Fields on Generative Artificial Intelligence (GenAI) Models requests and responses
+  description: >
+    Fields that capture the context of Generative Artificial Intelligence (GenAI) Models requests and responses
+
+    This field group definition is based on the Gen AI namespace of the OpenTelemetry Semantic Conventions (https://opentelemetry.io/docs/specs/semconv/attributes-registry/gen-ai/).
+
+  fields:
+    - name: gen_ai.agent.description
+      type: text
+      description: Free-form description of the GenAI agent provided by the application.
+      example: Helps with math problems; Generates fiction stories
+      level: extended
```

```suggestion
      level: extended
      beta: This field reuse is beta and subject to change.
```

Can you add `beta: This field reuse is beta and subject to change.` here, and to all the other new fields, to indicate all are in beta?